cogs/blacklist.py

```diff
@@ -2,16 +2,17 @@
 from discord import slash_command, option
 from discord.ext import commands
 from discord.ext.bridge import Bot
+
 from utilities.database import modifier
 
 
-class Blacklist(commands.Cog):
+class Blacklist(discord.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
     @slash_command()
     @commands.is_owner()
-    @option("server", discord.Guild, description="The server to blacklist")
```

Comment:
This was consciously chosen so we could just input the guild name. That actually works - as i just confirmed. Please dont change this.